app/models/container_node.rb

```diff
@@ -79,4 +79,18 @@ def ipaddress
   def cockpit_url
     URI::HTTP.build(:host => ipaddress, :port => 9090)
   end
+
+  def tenant_identity
+    if ext_management_system
+      ext_management_system.tenant_identity
+    else
+      User.super_admin.tap { |u| u.current_group = Tenant.root_tenant.default_miq_group }
+    end
+  end
+
+  def evaluate_alert(_alert_id, _event)
+    # currently only EmsEvents from hawkular are tested for node alerts,
+    # and these should automaticaly be translated to alerts.
+    true
```

Comment:
I plan to add something that will propegate to this point in https://github.com/ManageIQ/manageiq/pull/12773 then I will be able to add a check